main.go

```diff
@@ -325,6 +334,9 @@ func CreateExecutionPlan(cfg *Config) (Plan, error) {
 			if f.Gen2 {
 				args = append(args, "--gen2")
 			}
+			if f.HttpSecure != "" {
+				args = append(args, "--security-level="+f.security_level")
```

Comment:
Look at line 341, I think you should go the route of not using `=` but split them into separate array items.